pkgs/development/python-modules/textnets/default.nix

```diff
@@ -59,10 +64,18 @@ buildPythonPackage rec {
     "textnets"
   ];
 
-  disabledTests = [
-    # Test fails: A warning is triggered because of a deprecation notice by pandas.
-    # TODO: Try to re-enable it when pandas is updated to 2.1.1
-    "test_corpus_czech"
+  # The package has a few deprecation warnings that are thrown during testing (including as furure warnings).
+  #
+  # network.py:45 throws a UserWarning when it can't import Cythonized code.
+  # The .so files aren't present during the build, so let network.py use its pure-python replacement in testing.
+  # These files will be present when the package is installed.
```

Comment:
I guess we can add the following phase to avoid these warnings.

```nix
preCheck = ''
  cd $out
'';
```
For more information, see https://github.com/NixOS/nixpkgs/issues/255262

Reply:
Unfortunately, it doesn't seem to work. Adding the suggested change:

```nix
...
  preCheck = ''
    cd $ out
  '';
...
```

Yields the following error:

```console
...
Running phase: pytestCheckPhase
Executing pytestCheckPhase
/nix/store/3qnm3nwjajgqa771dmi2dnwxrw0kzq5m-stdenv-linux/setup: line 114: cd: too many arguments
/nix/store/3qnm3nwjajgqa771dmi2dnwxrw0kzq5m-stdenv-linux/setup: line 131: pop_var_context: head of shell_variables not a function context
```

Reply:
> ```
> ...
>   preCheck = ''
>     cd $ out
>   '';
> ...
> ```

Extra space is inserted. Please try `cd $out` since `out` is a variable.

Reply:
or try removing textnets from CWD

Reply:
When I correct preCheck to `cd $out`, build fails with:

```console
...
Running phase: pytestCheckPhase
Executing pytestCheckPhase
============================= test session starts ==============================
platform linux -- Python 3.11.8, pytest-7.4.4, pluggy-1.4.0
rootdir: /nix/store/qg9n8pnpmdyscy6df2kjyq07qvva0irc-python3.11-textnets-0.9.4
collecting ... ^Mcollected 0 items

============================ no tests ran in 0.00s =============================
/nix/store/3qnm3nwjajgqa771dmi2dnwxrw0kzq5m-stdenv-linux/setup: line 1578: pop_var_context: head of shell_variables not a function context
```

Reply:
Since this is a textnets build, I'm not sure what it would mean to remove it from CWD

Reply:
Found a fix for the underlying issue, see below.